java/com/google/copybara/git/AddExcludedFilesToIndex.java

```diff
@@ -68,13 +70,18 @@ void findSubmodules(Console console) throws RepoException {
   /**
    * Adds all the excluded files and submodules.
    */
-  void add() throws RepoException, IOException {
+  void add(Console console) throws RepoException, IOException {
```

Comment:
is passing the console to methods a pattern in copybara?

Reply:
Yeah it seems like that's how they do things